dulwich/server.py

```diff
@@ -232,18 +237,26 @@ def __init__(self, backend, args, proto, http_req=None,
         self.repo = backend.open_repository(args[0])
         self._graph_walker = None
         self.advertise_refs = advertise_refs
+        # A state variable for denoting that the have list is still
+        # being processed, and the client is not accepting any other
+        # data (such as side-band, see the progress method here).
+        self._processing_have_lines = False
+
+    @property
+    def _done_required(self):
```

Comment:
This should not be a property - we don't have properties for any of the other capabilities.